pallets/gear-builtin/src/benchmarking.rs

```diff
@@ -286,6 +290,24 @@ benchmarks! {
     } verify {
         assert!(ArkScale::<G2Affine>::decode(&mut &_result.unwrap()[..]).is_ok())
     }
+
+    staking_estimate_decode {
```

Comment:
nomination targets are limited by staking pallet
it could be used for benchmarks. moreover, it'd allow rejecting incoming requests by payload size if they're supposed to be invalid mb